agent/app/args/flag.go

```diff
@@ -21,9 +21,11 @@ const (
 	acceptInsecureCertUsage  = "Disable SSL certificate verification. We do not recommend setting this option"
 	licenseUsage             = "Print the LICENSE and NOTICE files and exit"
 	blacholeEC2MetadataUsage = "Blackhole the EC2 Metadata requests. Setting this option can cause the ECS Agent to fail to work properly.  We do not recommend setting this option"
+	capabilitiesUsage        = "Print the agent capabilities and exit"
```

Comment:
I'd change this to read 'print the capabilities given the current environment and agent' or something like that.

Reply:
How about this: `Prints the supported Agent capabilites based on its environment`